src/views/UnauthenticatedApp/__tests__/UnauthenticatedApp.test.tsx

```
@@ -0,0 +1,192 @@
+vi.mock('@/views/GameGuide', () => ({
+  default: function MockGameGuide() {
+    return <div data-testid="game-guide">Game Guide</div>;
+  },
+}));
+
+// Mock AuthDialog component
+vi.mock('@/components/auth/AuthDialog', () => ({
+  default: function MockAuthDialog({ open, close }: { open: boolean; close: () => void }) {
+    return open ? (
+      <div data-testid="auth-dialog">
+        <button onClick={close}>Close</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)